src/options/popup.js

```diff
@@ -70,7 +82,31 @@ export default {
         </li>`
       ).join("")
       + '</ul>',
-    save: `<button class="cc-btn cc-save">Save</button>`
+    save: `<button class="cc-btn cc-save">Save</button>`,
+    customize: `<button class="cc-btn cc-customize">{{customize}}</button>`,
+    customizeHeader: `<div class="cc-customize-header">{{customizeHeader}}</div>`,
+    customizeMessage: `<div class="cc-customize-message">{{customizeMessage}}</div>`,
+    acceptSelected: `<div class="cc-compliance">
+        <span class="sanddev-accept-selected">
+          <button class="cc-btn cc-save">{{acceptSelected}}</button>
+        </span>
+      </div>`,
+    customizeCategories: `<ul class="sanddev-categories">` +
+      categories.map(category =>
+        `<li class="cc-category ${category}">
+            <label>
+              <input type="checkbox" id="${category}" name="${category}"/>
+              <span class="cc-btn-checkbox">${categoriesDisplayNames[category]}</span>
+            </label>
+            <label class="sanddev-readmore" for="readmore-${category}">Read more</label>
+            <input id="readmore-${category}" type="checkbox"> 
+            <div class="sanddev-more-info">{{category${capitalizeFirstLetter(category)}}}</div>
```

Comment:
Please avoid runtime text manipulation like changing the first letter to uppercase - use a static map category -> some display name.

Reply:
this is not about display name - here variable name is generated like `categoryEssential` which is replaced later on by value of `categoryEssential`

Reply:
Please use a static mapping then: category -> categoryVariable like this:

```javascript
const VARIABLES = {
  [ESSENTIAL]: 'categoryEssential',
  [ANALYTICS]: 'categoryAnalytics`,
  ...etc
}
```